CHANGELOG-1.2.md

```diff
@@ -1,3 +1,12 @@
+# Changelog since v1.2.0
+
+## Notable Features
+
+* Adds CSI Migration support for Azure Disk/File, adds some backwards compatibility fixes for GCE PD Migration ([#292](https://github.com/kubernetes-csi/external-provisioner/pull/292), [@davidz627](https://github.com/davidz627))
```

Comment:
technically this PR https://github.com/kubernetes-csi/external-provisioner/pull/294 (?) I'm not clear on which CHANGELOGS go where and whether we refer to master branch or cherry picks to release branches

Reply:
Hm I guess it should use cherry-pick PRs instead of master ones.

Reply:
Fixed